src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class Circle extends Figure implements Drawable {
+    private final double radius;
+
+    public Circle(double radius,Color color) {
+        this.radius = radius;
+        this.color = color;
+        this.area = Math.pow(radius,2) * Math.PI;
```

Comment:
we won't be able to get access to area from a class in a different package, add method getArea()